changelog.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+- [Compiler Plugin Allows Passing an HTTP Listener with Configs to Listener Init](https://github.com/ballerina-platform/ballerina-standard-library/issues/2782)
```

Comment:
Make only the first letter capital

Reply:
Fixed with [fa56d50762b9878a8e7b1b49568d37da3bda2dd4](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/748/commits/fa56d50762b9878a8e7b1b49568d37da3bda2dd4)